diff --git a/node_modules/@walletconnect/universal-provider/dist/index.cjs.js b/node_modules/@walletconnect/universal-provider/dist/index.cjs.js
index 5cae6fe..9b61fe4 100644
--- a/node_modules/@walletconnect/universal-provider/dist/index.cjs.js
+++ b/node_modules/@walletconnect/universal-provider/dist/index.cjs.js
@@ -1,4 +1,4 @@
-"use strict";Object.defineProperty(exports,"__esModule",{value:!0});var Pa=require("@walletconnect/sign-client"),un=require("@walletconnect/utils"),Aa=require("@walletconnect/logger"),Ca=require("@walletconnect/jsonrpc-http-connection"),At=require("@walletconnect/jsonrpc-provider"),Rg=require("events");function $i(A){return A&&typeof A=="object"&&"default"in A?A:{default:A}}var Og=$i(Pa),Kt=$i(Ca),bg=$i(Rg);const Ia="error",Lg="wss://relay.walletconnect.com",Tg="wc",Dg="universal_provider",xa=`${Tg}@${2}:${Dg}:`,Ng="https://rpc.walletconnect.com/v1",ft={DEFAULT_CHAIN_CHANGED:"default_chain_changed"};var ge=typeof globalThis<"u"?globalThis:typeof window<"u"?window:typeof global<"u"?global:typeof self<"u"?self:{},Ui={exports:{}};/**
+"use strict";Object.defineProperty(exports,"__esModule",{value:!0});var Pa=require("@walletconnect/sign-client"),un=require("@walletconnect/utils"),Aa=require("@walletconnect/logger"),Ca=require("@walletconnect/jsonrpc-http-connection"),At=require("@walletconnect/jsonrpc-provider"),Rg=require("events");function $i(A){return A&&(typeof A=="object"||typeof A=="function")&&"default"in A?A:{default:A}}var Og=$i(Pa),Kt=$i(Ca),bg=$i(Rg);const Ia="error",Lg="wss://relay.walletconnect.com",Tg="wc",Dg="universal_provider",xa=`${Tg}@${2}:${Dg}:`,Ng="https://rpc.walletconnect.com/v1",ft={DEFAULT_CHAIN_CHANGED:"default_chain_changed"};var ge=typeof globalThis<"u"?globalThis:typeof window<"u"?window:typeof global<"u"?global:typeof self<"u"?self:{},Ui={exports:{}};/**
  * @license
  * Lodash <https://lodash.com/>
  * Copyright OpenJS Foundation and other contributors <https://openjsf.org/>
`